build.gradle

```
@@ -27,7 +27,7 @@ def pythonPluginForVersion = [
         "IC-183.4284.36": "PythonCore:2018.3.183.4284.36",
         "IC-191.6183.87": "PythonCore:2019.1.191.6183.53",
         "IC-192.5728.98": "PythonCore:2019.2.192.5728.98",
-        "LATEST-EAP-SNAPSHOT": "PythonCore:2019.2.192.5728.98",
+        "IC-193.4386.10-EAP-SNAPSHOT": "PythonCore:193.4386.10",
```

Newest version works fine. Let's leave these versions.
```suggestion
        "193.4697-EAP-CANDIDATE-SNAPSHOT": "PythonCore:193.4697.15",
```